nautobot/utilities/forms/forms.py

```diff
@@ -14,6 +14,8 @@
     "ImportForm",
     "ReturnURLForm",
     "TableConfigForm",
+    "PrefixFieldMixin",
+    "AddressFieldMixin",
```

Comment:
Let's keep this listing (and the mixin definitions themselves) in alphabetical order if we can, please.

Reply:
Can you move the actual class definitions to the appropriate order relative to the other classes as well?